coordinator/src/services/translation/request/dispense/dispense-notification.ts

```diff
@@ -203,14 +205,30 @@ function createDispenseNotificationSupplyHeaderPertinentInformation1(
     hl7UnitValue,
     hl7SuppliedLineItemQuantitySnomedCode
   )
+
+  const hl7RequestedLineItemSnomedCode = new hl7V3.SnomedCode(
+    requestedMedicationCoding.code,
+    requestedMedicationCoding.display
+  )
+  const hl7RequestedLineItemQuantitySnomedCode = new hl7V3.SnomedCode(
+    fhirMedicationDispense.contained[0].dispenseRequest.quantity.code,
```

Comment:
would you mind pulling `fhirMedicationDispense.contained[0]` out into a variable (and potentially using the `getMedicationDispenseContained()` function to get it)?

Reply:
talked about this, think we decided just to pull out [0] everywhere as it's a bit convoluted where each area of contained is accessed